common/src/paused_program_storage.rs

```diff
@@ -293,6 +295,7 @@ pub trait PausedProgramStorage: super::ProgramStorage {
                 static_pages: code.static_pages(),
                 state: ProgramState::Initialized,
                 expiration_block,
+                memory_infix: Default::default(),
```

Comment:
```suggestion
                memory_infix: session_id,
```

Also tests didn't catch this

Reply:
It is unnecessary at the moment since the rent logic is disabled. Will be changed in the subsequent PR(s)

Reply:
Unnecessary at the moment but required to be tested (it is enabled in mock)

Reply:
I think it is covered by pallet tests

Reply:
But tests didn't catch anything

Reply:
updated - https://github.com/gear-tech/gear/actions/runs/6609278198/job/17949186103?pr=3166#step:14:676

Reply:
It will be good to see infix asserts in the test as @breathx said